lib/src/components/pagination/Pagination.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+
+import { ColorScheme } from '../../experiments/color-scheme'
+import { Box } from '..'
+import {
+  RENDER_EIGHT_ELEMENTS,
+  RENDER_SIX_ELEMENTS
+} from './pagination.constants'
+import { PaginationProvider } from './pagination-context/PaginationContext'
+import { PaginationNextButton } from './PaginationNextButton'
+import { PaginationPages } from './PaginationPages'
+import { PaginationPreviousButton } from './PaginationPreviousButton'
+import { PaginationProps } from './types'
```

Comment:
```suggestion
import type { PaginationProps } from './types'
```

Please apply this if applicable elsewhere.

Reply:
yeah will update here and everywhere else